src/wikimedia/wikiFetch.ts

```
@@ -21,7 +38,9 @@ if (process.env.NODE_ENV !== 'development') {
     }, 60000).unref();
 }
 
-export async function wikiFetchGet(wiki: PapyrusWiki, path: `/${string}`): Promise<{}|null> {
+export const WIKI_FETCH_403FORBIDDEN: unique symbol = memoizeDevServerConst('WIKI_FETCH_403FORBIDDEN', () => Symbol('WIKI_FETCH_403FORBIDDEN')) as any;
+
+export async function wikiFetchGet(wiki: PapyrusWiki, path: `/${string}`): Promise<{}|null|typeof WIKI_FETCH_403FORBIDDEN> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)